Sources/OktaLogger/Analytics/Analytics.swift

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2020-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+import Foundation
+import OktaLogger
+
+extension OktaLogger {
```

Comment:
Should it be really in Logger namespace? Why it can't live as a separate class?

Reply:
Since we have separated  Analytics use case and functionality, we can go with another name space  OktaAnalytics